Tests/scripts/hook_validations/integration.py

```diff
@@ -181,6 +187,33 @@ def is_changed_subtype(self):
 
         return self._is_valid
 
+    def is_valid_beta(self, is_new=False):
+        """Validate that that beta integration has correct beta attributes"""
+        beta = self.current_integration.get('beta', False)
+        if not beta:
+            print_error("Beta integration yml file should have the field \"beta: true\", but was not found"
+                        " in the file {}".format(self.file_path))
+            self._is_valid = False
```

Comment:
better to extract into functions, it's way less readable now then if it was:
self._is_valid = not All(self.has_beta_param(), 
                                     self.check_beta_display_name...

Reply:
done